packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -77,81 +79,88 @@ const EDiscovery = WebexPlugin.extend({
+      timeout: timeoutMs
     });
   },
 
   @waitForValue('@')
   /**
    * Retrieves all the compliance officers reports
-   * @param {int} offset - Start position from which to retrieve records
-   * @param {int} size - Number of records to retrieve
+   * @param {Object} {offset, size} - Start position from which to retrieve records and number of records to retrieve
+   * @param {int} timeoutMs - optional connection timeout in milliseconds, defaults to 10s
    * @returns {Promise<ResponseEntity<Array<ReportRecord>>>} Http Response containing a list of report records
    */
-  getReports({offset, size} = {}) {
+  getReports({offset, size} = {}, timeoutMs = 10000) {
     return this.request({
       method: 'GET',
       service: 'ediscovery',
       resource: 'reports',
-      qs: {offset, size}
+      qs: {offset, size},
+      timeout: timeoutMs
     });
   },
 
   @waitForValue('@')
   /**
    * Deletes a specified report
    * @param {UUID} reportId - Id of the report being requested for deletion
+   * @param {int} timeoutMs - optional connection timeout in milliseconds, defaults to 10s
    * @returns {Promise<ResponseEntity>} HttpResponse indicating if delete was successful
    */
-  deleteReport(reportId) {
+  deleteReport(reportId, timeoutMs = 10000) {
     if (!reportId) {
       throw Error('Undefined parameter');
     }
 
     return this.request({
       method: 'DELETE',
       service: 'ediscovery',
-      resource: `reports/${reportId}`
+      resource: `reports/${reportId}`,
+      timeout: timeoutMs
     });
   },
 
   @waitForValue('@')
   /**
    * Restarts a completed or cancelled report so that it begins again from scratch
    * @param {UUID} reportId - Id of the report being requested
+   * @param {int} timeoutMs - optional connection timeout in milliseconds, defaults to 10s
    * @returns {Promise<ResponseEntity<ReportRecord>>} Http response containing the report record
    */
-  restartReport(reportId) {
+  restartReport(reportId, timeoutMs = 10000) {
     if (!reportId) {
       throw Error('Undefined parameter');
     }
 
     return this.request({
       method: 'PUT',
       service: 'ediscovery',
-      resource: `reports/${reportId}`
+      resource: `reports/${reportId}`,
+      timeout: timeoutMs
     });
   },
 
   @waitForValue('@')
   /**
    * Retrieves content associated with a report
    * @param {UUID} reportId - Id of the report which contains the content
-   * @param {int} offset - Start position from which to retrieve content
-   * @param {int} size - Number of activities to retrieve
+   * @param {Object} {offset, size} - Start position from which to retrieve content and number of activities to retrieve
+   * @param {int} timeoutMs - optional connection timeout in milliseconds, defaults to 10s
    * @returns {Promise<ResponseEntity<[Activity]>>} Http response containing the activities
    */
-  getContent(reportId, {offset, size} = {}) {
+  getContent(reportId, {offset, size} = {}, timeoutMs = 10000) {
```

Comment:
Given the current issue with getContent delays I assume you'll be specifying an non-default value in the downloader

Reply:
the timeout in the request module is for connection and first byte transferred, if the overall transfer of data takes > 10s then that's ok, it won't be killed by the timeout